models/pagination-1.0.0.yaml

```diff
@@ -1,42 +1,42 @@
-Pagination:
```

Comment:
Is it the line endings that have changed here? I can't really see any differences in this file

Reply:
Yes. Somehow the line endings were messed up. 
I have updated the file and now there are no more diffs.